### PR TITLE
ns-3 3.36

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

File: Formula/ns-3.rb [hunk 1]
```diff
@@ -25,43 +25,36 @@ class Ns3 < Formula
...
            "-lns#{version}-core", "-lns#{version}-network", "-lns#{version}-internet",
            "-lns#{version}-point-to-point", "-lns#{version}-applications",
-           "-std=c++11", "-o", "test"
+           "-o", "test"
     system "./test"
 
-    system Formula["python@3.10"].opt_bin/"python3", pkgshare/"first.py"
+    # As of ns-3.36, the python bindings do not work with python 3.9 or python 3.10
+    # https://gitlab.com/nsnam/ns-3-dev/-/blob/master/doc/manual/source/python.rst#L46-56
+    # system Formula["python@3.10"].opt_bin/"python3", pkgshare/"first.py"
```
Comment (SMillerDev) [inline Formula/ns-3.rb:58]: From that file:

> Compiling the existing bindings should work through Python 3.10.

Only:

> The API scanning framework (in particular, pygccxml) requires Python 3.6 through 3.8 but is not compatible with Python 3.9 or 3.10.

Comment (chenrui333) [reply Formula/ns-3.rb:58]: raise [a question](https://gitlab.com/nsnam/ns-3-dev/-/issues/643) about this.

Comment (TommyPec) [reply Formula/ns-3.rb:58]: To clarify... we did test the existing bindings with Python up to 3.10, and they should work.

If you need to re-generate the bindings you'll need Python 3.8, but regenerating the bindings is only required if you modify the ns-3 sources in `contrib` or `src`.

The problem (as pointed out in our tracker) is that you need to add `--enable-python-bindings` to the configuration.

As a side note, I'd avoid to use `cmake` directly, there's a nice build and configuration script that handles everything. It's named `ns3`.

**Comment by TommyPec** [comment]
I'd kindly ask @chenrui333 to revive this (perhaps pointing to ns-3.36.1).

If the python bindings are a major issue, I'd suggest to configure ns-3 *without* them. Right now they're not stable for Linux either, so... we're working on a solution - in the meantime disable them.

**Pull request closed without merging**